sechub-sereco/src/test/java/com/daimler/sechub/sereco/importer/SarifV1JSONImporterTest.java

```diff
@@ -0,0 +1,170 @@
+    private static String sarifEmptyResult;
+    private static String sarifThreadflowsExample;
+    
+    @BeforeClass
+    public static void before() {
+        importerToTest = new SarifV1JSONImporter();
+        sarifBrakeman = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_brakeman.json");
+        sarifEmptyResult = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_empty_results.json");
+        sarifThreadflowsExample = SerecoTestFileSupport.INSTANCE.loadTestFile("sarif/sarif_2.1.0_threadflows_example.json");
+    }
+
+    @Test
+    public void brakeman_sarif_report_can_be_imported() {
+        /* prepare */
+
+        ImportParameter paramBrakeman = ImportParameter.builder().importData(sarifBrakeman).importId("id1").productId("SARIF").build();
+        
+        /* execute */
+        ProductImportAbility ableToImportBrakemanSarif = importerToTest.isAbleToImportForProduct(paramBrakeman);
+        
+        /* test */
+        assertEquals("Was able to import sarif!", ProductImportAbility.ABLE_TO_IMPORT, ableToImportBrakemanSarif);    
+    }
+    
+    @Test
+    public void threadflow_sarif_report_can_be_imported() {
+        /* prepare */
+
+        ImportParameter paramThreadFlows = ImportParameter.builder().importData(sarifThreadflowsExample).importId("id1").productId("SARIF").build();
+
+        /* execute */
+        ProductImportAbility ableToImportThreadFlowSarif = importerToTest.isAbleToImportForProduct(paramThreadFlows);
+
+        /* test */
+        assertEquals("Was able to import sarif!", ProductImportAbility.ABLE_TO_IMPORT, ableToImportThreadFlowSarif);
+    }
+
+
+    @Test
+    public void empty_sarif_report_throws_exception() {
+        
+        /* test */
+        assertThrows(IOException.class, () -> {
+            importerToTest.importResult("");
+        });
+    }
+    
+    @Test
+    public void null_sarif_report_throws_exception() {
+        
+        /* test */
+        assertThrows(IOException.class, () -> {
+            importerToTest.importResult(null);
+        });
+    }
+    
+    @Test
+    public void sarif_report_has_errorlevel() throws Exception {
+        /* prepare */
+        SerecoMetaData result = importerToTest.importResult(sarifBrakeman);
+
+        /* execute */
+        List<SerecoVulnerability> vulnerabilities = result.getVulnerabilities();
+        SerecoVulnerability vulnerability = vulnerabilities.get(0);
+
+        /* test */
+        assertEquals(SerecoSeverity.HIGH, vulnerability.getSeverity());
+    }
+
+    @Test
+    public void sarif_report_has_no_description() throws Exception {
+        /* prepare */
+        SerecoMetaData result = importerToTest.importResult(sarifBrakeman);
+
+        /* execute */
+        List<SerecoVulnerability> vulnerabilities = result.getVulnerabilities();
+        SerecoVulnerability vulnerability = vulnerabilities.get(0);
+
+        /* test */
+        assertEquals("", vulnerability.getDescription());
```

Comment:
This test only aims to ensure and stress that the description field is empty, because I adapted the behavior of `CheckmarxV1XMLImporter.java`. If we would define an actual SARIF field to be matched to the description field this test would be obsolete.